tests/integration/test_benchmark.py

```diff
@@ -301,21 +301,24 @@ def test_benchmark_single_table_timeout():
 
     # Assert
     assert total_time < 50.0  # Buffer time for code not in timeout
-    expected_scores = pd.DataFrame({
-        'Synthesizer': {0: 'GaussianCopulaSynthesizer'},
-        'Dataset': {0: 'insurance'},
-        'Dataset_Size_MB': {0: 3.340128},
-        'Train_Time': {0: None},
-        'Peak_Memory_MB': {0: None},
-        'Synthesizer_Size_MB': {0: None},
-        'Sample_Time': {0: None},
-        'Evaluate_Time': {0: None},
-        'Diagnostic_Score': {0: None},
-        'Quality_Score': {0: None},
-        'Privacy_Score': {0: None},
-        'error': {0: 'Synthesizer Timeout'},
-    })
-    pd.testing.assert_frame_equal(scores, expected_scores)
+    timeout_scores = pd.Series(
```

Comment:
Some of these tests don't seem related to the UniformSynthesizer changes, why did you update them?